autopush/tests/test_web_base.py

```diff
@@ -57,8 +58,11 @@ def setUp(self):
                                  headers={"ttl": "0"},
                                  host='example.com:8080')
 
+        mock_db = Mock()
```

Comment:
I have a test_db() in support you can also use, but it's not important here/this is fine